DeeployTest/generateNetwork.py

```diff
@@ -90,11 +122,21 @@
 
     platform, signProp = mapPlatform(args.platform)
 
-    for index, num in enumerate(test_inputs):
+    for index, name in enumerate(inputs.files):
         # WIESP: Do not infer types and offset of empty arrays
+        num = test_inputs[index]
         if np.prod(num.shape) == 0:
             continue
-        _type, offset = inferInputType(num, signProp)[0]
+        num = test_inputs[index]
```

Comment:
There seems to be some code duplication, and line 127 fails in the `cortexm-models` and `generic-models` tests. Please check.

Reply:
[cc8f8ec](https://github.com/pulp-platform/Deeploy/pull/68/commits/cc8f8ecd81987bb0259e22cc981f137c643f725c) should do the trick